app/src/main/java/com/orgzly/android/util/OrgFormatter.kt

```diff
@@ -53,6 +54,8 @@ object OrgFormatter {
 
     private val CHECKBOXES_PATTERN = Pattern.compile("""^\s*-\s+(\[[ X]])""", Pattern.MULTILINE)
 
+    private val FULL_TIMESTAMP ="([\\[<][0-9]{4,}-[0-9]{2}-[0-9]{2} +[A-Za-z\\.]* +[0-9]{1,2}:[0-9]{2}[\\]>])"
```

Comment:
Maybe we can call it `INACTIVE_DATETIMERANGE`, as it's made of [inactive timestamps](https://www.gnu.org/software/emacs/manual/html_node/org/Timestamps.html) denoting a date-time range? 
```suggestion
    private val INACTIVE_DATETIMERANGE ="([\\[<][0-9]{4,}-[0-9]{2}-[0-9]{2} +[A-Za-z\\.]* +[0-9]{1,2}:[0-9]{2}[\\]>])"
```

Reply:
And a patch/diff of my suggestions for `OrgFormatter.kt` just in case :slightly_smiling_face:   
```diff
diff --git a/app/src/main/java/com/orgzly/android/util/OrgFormatter.kt b/app/src/main/java/com/orgzly/android/util/OrgFormatter.kt
index badcad64..c98d99e6 100644
--- a/app/src/main/java/com/orgzly/android/util/OrgFormatter.kt
+++ b/app/src/main/java/com/orgzly/android/util/OrgFormatter.kt
@@ -54,7 +54,7 @@ object OrgFormatter {
 
     private val CHECKBOXES_PATTERN = Pattern.compile("""^\s*-\s+(\[[ X]])""", Pattern.MULTILINE)
 
-    private val FULL_TIMESTAMP ="([\\[<][0-9]{4,}-[0-9]{2}-[0-9]{2} +[A-Za-z\\.]* +[0-9]{1,2}:[0-9]{2}[\\]>])"
+    private val INACTIVE_DATETIME_RANGE ="([\\[<][0-9]{4,}-[0-9]{2}-[0-9]{2} +[A-Za-z\\.]* +[0-9]{1,2}:[0-9]{2}[\\]>])"
 
     private const val FLAGS = Spanned.SPAN_EXCLUSIVE_EXCLUSIVE
 
@@ -395,7 +395,7 @@ object OrgFormatter {
             val start = m.start(2) // Content start
             val end = m.end()
 
-            val CLOCKED_TIMES_P = Pattern.compile("(CLOCK: *" + FULL_TIMESTAMP + ") *(-- *" + FULL_TIMESTAMP + ")?( *=> *[0-9]{1,4}:[0-9]{2})?[\\r\\n]*")
+            val CLOCKED_TIMES_P = Pattern.compile("(CLOCK: *$INACTIVE_DATETIME_RANGE) *(-- *$INACTIVE_DATETIME_RANGE)?( *=> *[0-9]{1,4}:[0-9]{2})?[\\r\\n]*")
 
             val m2 = CLOCKED_TIMES_P.matcher(content?.subSequence(start, end))
             while (m2.find()) {
@@ -456,7 +456,7 @@ object OrgFormatter {
                 // If we find an already clocked entry, we clock-out
                 if(pos.isNotEmpty()) {
                     // Find the timestamp to compute the difference
-                    val m2 = Pattern.compile(FULL_TIMESTAMP).matcher(content)
+                    val m2 = Pattern.compile(INACTIVE_DATETIME_RANGE).matcher(content)
 
                     if (m2.find()) {
                         val clock_in = OrgDateTime.parseOrNull(m2.group(1))
```

Reply:
Thanks for the comments ! I've updated the code according to the review.    
I actually didn't use your suggested name `INACTIVE_DATETIME_RANGE` because the associated regex only represents one timestamp and not the range (`CLOCKED_TIMES_P` represents the whole CLOCK entry with the timerange).    
I also had to update the regex to only parse inactive timestamps (The previous one parsed both active and inactive ones).    
Please close the discussion, if you are ok with the changes.

Reply:
Using `INACTIVE_DATETIME` is even better!
It seems like the way to mark a discussion as resolved is by hiding the comment, so I'll do that.

Reply:
Thanks !
Normally, you should have a `Resolve conversation` button at the end of the thread of your comment to do so. It should allow you to mark the discussion as done.
I have it on my side, don't you ? Or maybe those must be resolved by the maintainer ? :thinking:

Reply:
Nope, don't have the Resolve conversation button on my side. Maybe you or Nevin can resolve this issue?

Reply:
I'll resolve the conversation then !